src/resources/addon.ts

```diff
@@ -74,7 +74,7 @@ export class Addon extends Model {
     }, ChargeBee._env)
   }
 
-  public static list(params?: _addon.addon_list_params):RequestWrapper {
+  public static list(params?: _addon.addon_list_params):RequestWrapper<Addon> {
```

Comment:
This should return a list i believe (in the very specific chargebee format with the next_offset string (why string? why?))

Reply:
this goes for all the lists. Also some endpoints return multiple resources, like a subscription and a customer and bunch of other optional stuff.